disputer/disputer.js

```diff
@@ -0,0 +1,84 @@
+const { Logger } = require("../financial-templates-lib/Logger");
+
+class Disputer {
+  constructor(expiringMultiPartyClient, account) {
+    this.account = account;
+
+    // Expiring multiparty contract to read contract state
+    this.empClient = expiringMultiPartyClient;
+
+    // Instance of the expiring multiparty to perform on-chain liquidations
+    this.empContract = this.empClient.emp;
+  }
+
+  // Queries disputable liquidations and disputes any that were incorrectly liquidated.
+  queryAndDispute = async priceFunction => {
+    Logger.debug({
+      at: "Disputer",
+      message: "Checking for any disputable liquidations"
+    });
+
+    // Update the client to get the latest liquidation information.
+    await this.empClient._update();
+
+    // Get the latest disputable liquidations from the client.
+    const undisputedLiquidations = this.empClient.getUndisputedLiquidations();
+    const disputeableLiquidations = undisputedLiquidations.filter(liquidation =>
+      this.empClient.isDisputable(liquidation, priceFunction(liquidation.liquidationTime))
+    );
+
+    if (disputeableLiquidations.length === 0) {
+      Logger.debug({
+        at: "liquidator",
+        message: "No disputable liquidations"
+      });
+
+      // Nothing left to do, so return.
+      return;
+    }
+
+    Logger.info({
+      at: "Disputer",
+      message: "Disputable liquidation(s) detected!",
+      number: disputeableLiquidations.length,
+      underCollateralizedPositions: disputeableLiquidations
```

Comment:
nit: we can name the key's in this object to anything we'ld like. Should this perhaps be:
```suggestion
      disputeableLiquidations: disputeableLiquidations
```